rest_framework/fields.py

```diff
@@ -653,6 +654,30 @@ def to_representation(self, value):
         return str(value)
 
 
+class IPAddressField(CharField):
+    """Support both IPAddressField and GenericIPAddressField"""
+
+    default_error_messages = {
+        'invalid': _('Enter a valid IPv4 or IPv6 address.'),
+    }
+
+    def __init__(self, protocol='both', unpack_ipv4=False, **kwargs):
+        self.protocol = protocol
```

Comment:
Let's lowercase the user input here. `self.protocol = protocol.lower()` - that'll be marginally more robust.